### PR TITLE
Created a CSS Grid Masonary Layout

### PR DESCRIPTION
**Pull request by AbbasShaikh0304**

### CODE REVIEW

File: tasks-by-users/AbbasShaikh0304/06-CSS Grid- Masonary/style.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+  max-width: 100px;
+
+  margin-bottom: 10px;
+}
+.container .box:nth-child(1) {
+  grid-column: span 2;
+  grid-row: span 1;
+}
+.container .box:nth-child(2) {
```
Comment (AlishaPrasad) [inline tasks-by-users/AbbasShaikh0304/06-CSS Grid- Masonary/style.css:57]:
```suggestion
.container .box:nth-child(2),
.container .box:nth-child(4) {
```

File: tasks-by-users/AbbasShaikh0304/06-CSS Grid- Masonary/style.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  grid-column: span 1;
+  grid-row: span 2;
+}
+.container .box:nth-child(4) {
+  grid-column: span 1;
+  grid-row: span 2;
+}
```
Comment (AlishaPrasad) [inline tasks-by-users/AbbasShaikh0304/06-CSS Grid- Masonary/style.css:64]: remove

Comment (AbbasShaikh0304) [reply tasks-by-users/AbbasShaikh0304/06-CSS Grid- Masonary/style.css]: Thankyou for you suggestions. Commited the Changes @AlishaPrasad

File: tasks-by-users/AbbasShaikh0304/06-CSS Grid- Masonary/style.css (hunk 1 above)
Comment (AbbasShaikh0304) [reply tasks-by-users/AbbasShaikh0304/06-CSS Grid- Masonary/style.css]: Thankyou for you suggestions. Commited the Changes @AlishaPrasad

**Review by shaikh-7abish — APPROVED** [review]

File: tasks-by-users/AbbasShaikh0304/06-CSS Grid- Masonary/style.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
+* {
+  margin: 0%;
+  padding: 0%;
```
Comment (Tanvir04khan) [inline tasks-by-users/AbbasShaikh0304/06-CSS Grid- Masonary/style.css:3]:
```suggestion
 margin:0;
  padding: 0;
```

Comment (AbbasShaikh0304) [reply tasks-by-users/AbbasShaikh0304/06-CSS Grid- Masonary/style.css:3]: Commited Suggested changes @Tanvir04khan

**Review by coder-mezab — APPROVED** [review]

**Review by nasir-k — APPROVED** [review]

**Pull request merged**